File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] pair=new int[2];
        
        for(int i=0;i<nums.length-1;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    pair[0]=i;
                    pair[1]=j;
                    break;
                }
            }
        }
        return pair;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        for i in range(len(nums)):
            exists = target-nums[i]
            if exists in hashmap:
                return [i, hashmap[exists]]
            hashmap[nums[i]] = i
        